app/containers/AudioPlayer/index.test.tsx

```
@@ -0,0 +1,656 @@
+	});
+
+	describe('InteractionManager task cleanup', () => {
+		it('cancels the InteractionManager task when the component unmounts', async () => {
+			const { unmount } = render(<AudioPlayer {...defaultProps} />);
+
+			act(() => unmount());
+
+			expect(mockTaskCancel).toHaveBeenCalled();
+		});
+
+		it('does not create a task (and therefore never cancels) when conditions are not met', async () => {
```

ESLint

[error] Async arrow function has no 'await' expression.

(require-await)